### PR TITLE
bug-fix

### PR DESCRIPTION
**Pull request by Cloudy1233**

### CODE REVIEW

**Comment by rmoldo** [comment]
The validation script is not working on the addUser.jsp. Do not merge yet

**Comment by Cloudy1233** [comment]
> not merge yet
La mine functioneaza, iti da ceva eroare sau ?

**Comment by rmoldo** [comment]
> > La mine functioneaza, iti da ceva eroare sau ?
Da nu mai mergea scriptul de validare asa ca am rezolvat eu in ultimul commit

File: src/main/webapp/WEB-INF/pages/deleteUser.jsp [hunk 1]
```diff
@@ -9,43 +9,49 @@
...
+                    <th scope="col" class="border-0 text-uppercase font-medium">First Name</th>
+                    <th scope="col" class="border-0 text-uppercase font-medium">Username</th>
+                    <th scope="col" class="border-0 text-uppercase font-medium">Role</th>
+                </tr>
+            </tbody>
+            <input form="form_2" type="hidden">
+            <button class="btn btn-danger" type="submit" form="form_2">Delete Users</button>
+            <c:forEach var="user" items="${listDeleteUsers}" varStatus="status">
+                <tbody>
+                    <tr>
```
Comment (dragosdinescu) [inline src/main/webapp/WEB-INF/pages/deleteUser.jsp:40]: tr tag is not closed

File: src/main/webapp/WEB-INF/pages/users.jsp [hunk 2]
```diff
@@ -11,36 +11,42 @@
...
-                                ${user.username}
-                            </td>
-                            <td>
-                                ${user.email}
-                            </td>
-                            <td>
-                                ${user.position}
-                            </td>
-    </c:forEach>
+            </c:forEach>
```
Comment (dragosdinescu) [inline src/main/webapp/WEB-INF/pages/users.jsp:49]: wrong order of closing tags, it should be like this

`    </tbody>
</c:forEach>`

**Pull request merged**